docs/usage/configuration.rst

```diff
@@ -90,3 +90,12 @@ Set of fields that allow you to generate a collection with random lengths. By de
 .. literalinclude:: /examples/configuration/test_example_6.py
     :caption: Randomized collection length
     :language: python
+
+Allow None Optionals
+----------------------------
+
+Allow None to be set for optional types. By default, Optional type will be ignored.
```

Comment:
Could you elaborate on this to explain that the default for `__allow_none_optionals__` is `True` in which case, `None` could be returned and that when it's set to `False`, then whatever `T` is in `Optional[T]` will be returned?

Reply:
I just wanted to point out that, I think it would be a good idea to emphasize that setting `__allow_none_optionals__` to `True` does not guarantee that `None` will be set for fields with `Optional[T]`, but that it is a possible value.